meson.build

```diff
@@ -389,7 +389,7 @@ luaold_interpreters = [
 
 lua_pc_name = ''
 
-if sys_windows or get_option('lua-interpreter') == 'lua'
+if get_option('lua-interpreter') == 'lua'
   if sys_windows and get_option('lua-interpreter') != 'lua'
     warning('There is no "' + get_option('lua-interpreter')
             + '" binding option for Windows, backing up to "lua"')
```

Comment:
Is this still needed?

Reply:
Yes, it is showing a warning in the case of someone not changing the default `lua-interpter` (that is `luajit`) to `lua`.

Reply:
But if that happens there is no way he enters this `if`

Reply:
oooh thanks, I'll fix that